cartography/intel/aws/rds.py

```diff
@@ -216,6 +216,114 @@ def load_rds_instances(
     _attach_clusters(neo4j_session, clusters, aws_update_tag)
 
 
+@timeit
+@aws_handle_regions
+def get_rds_snapshot_data(boto3_session: boto3.session.Session, region: str) -> List[Any]:
+    """
+    Create an RDS boto3 client and grab all the DBSnapshots.
+    """
+    client = boto3_session.client('rds', region_name=region)
```

Comment:
Non-blocker: try to use the `aws_paginate` helper function.
https://github.com/lyft/cartography/blob/master/cartography/util.py#L138
```
snapshots = aws_paginate(client, 'describe_db_snapshots', 'DBSnapshots')
```

Reply:
thx